rust/otap-dataflow/configs/fake-parquet.yaml

```diff
@@ -22,7 +22,13 @@ nodes:
     kind: exporter
     plugin_urn: "urn:otel:otap:parquet:exporter"
     config:
-      base_uri: /tmp
+      storage:
+        # file:
+        #   base_uri: /home/jakedern/local-fabric-test
+        azure:
+          base_uri: https://barquettest.blob.core.windows.net/barquet1
+          auth:
+            type: AzureCli
```

Comment:
Config looks good! This seems like a more extensible way to configure the the storage instead of trying to infer it from the path as suggested in #501